app/presenters/licences/view-licence-communications.presenter.js

```diff
@@ -31,7 +31,7 @@ function _communications(communications, documentId, licenceId) {
       link: _link(communication.id, documentId, licenceId),
       type: _type(communication),
       sender: communication.event.issuer,
-      sent: formatLongDate(new Date(communication.event.createdAt)),
+      sent: formatLongDate(new Date(communication.createdAt)),
```

Comment:
this can be the event created at but it might not align with the notification created at / send after date.